scripts/drag-n-bar-form-manager.js

```diff
@@ -147,10 +147,20 @@
         head.appendChild(fullscreenButton);
       }
 
-      window.addEventListener('resize', self.updateFormResponsiveness);
+      const onResize = () => {
+        const screenWasResized = this.lastWindowWidth !== window.innerWidth || this.lastWindowHeight !== window.innerHeight;
+        if (screenWasResized) {
+          this.updateFormResponsiveness();
+
+          this.lastWindowWidth = window.innerWidth;
+          this.lastWindowHeight = window.innerHeight;
+        }
+      };
```

Comment:
This is not actually doing anything for or against the bug, but is a nice guard for an extra performance improvement